File: PlumsailWebApp/ClientApp/src/selectors/resume.js
import moment from 'moment';
import _ from 'lodash';

const createResume = () => ({
  id: 0,
  name: '',
  placement: '',
  phone: '',
  email: '',
  experience: 0,
  isRelocate: false,
  position: '',
  createAt: moment().toISOString(false)
});

const normalizeMoment = value => {
  let normalizedValue = _.isNull(value) ? value : _.defaultTo(value, moment());
  if (!_.isNull(normalizedValue)) {
    normalizedValue = _.isObjectLike(normalizedValue) ? normalizedValue : moment(normalizedValue);
    if (!_.isFunction(normalizedValue.isValid) || !normalizedValue.isValid()) {
      normalizedValue = moment();
    }
  }
  return normalizedValue;
};

export default Object.freeze({
  createResume,
  normalizeMoment
});

File: PlumsailWebApp/Models/Resume.cs
using System;

namespace PlumsailWebApp.Models
{
    public class Resume : BaseEntity
    {
        public string Name { get; set; }
        
        public string Placement { get; set; }
        
        public string Phone { get; set; }
        
        public string Email { get; set; }
        
        public decimal Experience { get; set; }
        
        public bool IsRelocate { get; set; }
        
        public string Position { get; set; }
        
        public DateTime CreateAt { get; set; }
        
        public string PhotoBase64 { get; set; }


        public Resume()
        {
        }
    }
}